### PR TITLE
accept relative timestamps from canonical format

### PR DESCRIPTION
**Pull request by jnunyez**
This PR  add the capability to calculate relative timestamps when data input comes in canonical format. To be used for normalizing the timescales for the x-axis for plotting the timeseries of graphs.

### CODE REVIEW

File: src/vse_sync_pp/parsers/parser.py [hunk 1]
```diff
@@ -83,21 +92,23 @@ def parse(self, file, relative=False):
...
                 if relative:
-                    timestamp = getattr(parsed, 'timestamp', None)
-                    if timestamp is not None:
-                        if tzero is None:
-                            tzero = timestamp
-                        parsed = parsed._replace(timestamp=timestamp - tzero)
-                yield parsed
-    def canonical(self, file):
+                    tzero, parsed = relative_timestamp(parsed, tzero)
+            yield parsed
```
Comment (nocturnalastro) [inline src/vse_sync_pp/parsers/parser.py:96]: In both cases this needs to be indented by one level to allow the same behaviour as before

**Comment by nocturnalastro** [comment]
> I guess that each plot will be relative to _its own_ t_zero. How do we communicate this?
Could just say `relative time` instead of just `time`

File: src/vse_sync_pp/parsers/parser.py (hunk 1 above)
Comment (jnunyez) [reply src/vse_sync_pp/parsers/parser.py]: fixed.

**Comment by jnunyez** [comment]
> I guess that each plot will be relative to _its own_ t_zero. How do we communicate this?
Yes, in the plots instead of timestamp we can just say "elapsed time [s]" wdyt? We also need to communicate the units:
Elapsed time [s] vs Time Error (Unfiltered) [ns]

**Review by spenced — APPROVED** [review]
Propose we address communicating t_zero/relative time to the test cases and test report (the consumers of this code).

**Pull request merged**